eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelData.scala

```diff
@@ -412,7 +412,10 @@ object RealScidStatus {
  */
 case class ShortIds(real: RealScidStatus, localAlias: Alias, remoteAlias_opt: Option[Alias])
 
-sealed trait LocalFundingStatus { def signedTx_opt: Option[Transaction] }
+sealed trait LocalFundingStatus {
+  def signedTx_opt: Option[Transaction]
+  def localSigs_opt: Option[TxSignatures]
```

Comment:
Just a 📃 to future self
```suggestion
  /** We store local signatures for the purpose of retransmitting if the funding/splicing flow is interrupted. */
  def localSigs_opt: Option[TxSignatures]
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2748/commits/4f2e481e5dd920d1c108c3fc0c4a1c73dd90aed7